DESCRIPTION

```diff
@@ -19,6 +19,7 @@ Imports:
     glue,
     hardhat,
     parsnip (>= 0.0.4),
+    utils
```

Comment:
Missing comma after utils?

Reply:
Actually I think I fixed this already, if you remove utils from Imports and remove the ImportsFrom for `capture.output()` it should work